crates/holochain_conductor_api/src/admin_interface.rs

```diff
@@ -548,6 +555,9 @@ pub enum AdminResponse {
     /// The successful response to an [`AdminRequest::GrantZomeCallCapability`].
     ZomeCallCapabilityGranted,
 
+    /// The successful response to an [`AdminRequest::CapabilityGrantInfo`].
+    ZomeCallCapabilityGrantInfo(AppCapGrantInfo),
```

Comment:
We clearly haven't been great at following a pattern here, but the intention has been to end with a verb on these return variants. Perhaps `CapabilityGrantsListed` here and `ListCapabilityGrants` above? Just to make a clear pair, because these have slightly different names